wca-guidelines.md

```diff
@@ -224,7 +224,7 @@ To be more informative, each Guideline is classified using one of the following
 
 ## <article-H><multiple-blindfolded><multipleblindfoldedsolving> Article H: Multiple Blindfolded Solving
 
-- H1+) [ADDITION] If a duplicate puzzle is found during an attempt, the cube puzzle be re-scrambled using an unused scramble, at the discretion of the WCA Delegate.
+- H1+) [ADDITION] If a duplicate puzzle is found during an attempt, the puzzle may be re-scrambled using a not used scramble for the attempt, at the discretion of the WCA Delegate.
```

Comment:
Hmm, I think "unused" is sufficiently clear and simpler, so I would prefer to keep that.
```suggestion
- H1+) [ADDITION] If a duplicate puzzle is found during an attempt, the puzzle may be re-scrambled using an unused scramble, at the discretion of the WCA Delegate.
```

Reply:
I agree, everything else LGTM